src/lib/core/trigger.cc

```diff
@@ -35,6 +35,8 @@
 
 #include <small/region.h>
 
+#include <small/mempool.h>
```

Comment:
Nit: redundant empty line between the two includes.

Reply:
The whole file's like that. Fixed it all:
<details><summary>diff</summary>

```diff
diff --git a/src/lib/core/trigger.cc b/src/lib/core/trigger.cc
index d714cbb6b..9a53c9284 100644
--- a/src/lib/core/trigger.cc
+++ b/src/lib/core/trigger.cc
@@ -30,11 +30,8 @@
  */
 
 #include "trigger.h"
-
 #include "fiber.h"
-
 #include <small/region.h>
-
 #include <small/mempool.h>
 
 static void

```

</details>